packages/core/src/shared/db/chatDb/chatDb.ts

```diff
@@ -229,23 +253,29 @@ export class Database {
     addMessage(tabId: string, tabType: TabType, conversationId: string, message: Message) {
         if (this.initialized) {
             const tabCollection = this.db.getCollection<Tab>(TabCollection)
+            this.logger.info(`Adding message: tabId=${tabId}, tabType=${tabType}, conversationId=${conversationId}`)
 
             let historyId = this.historyIdMapping.get(tabId)
 
             if (!historyId) {
                 historyId = crypto.randomUUID()
+                this.logger.info(`No historyId found, creating new one: ${historyId}`)
```

Comment:
Should some of these info-level logs be bumped down? Is it noisy when running locally?

Reply:
Bumped a few down. Will bump more down if the logs become too noisy